Cards/MadGraph5_aMCatNLO/Others/GJets-4Jets_dRGJ-0p2_HT-100to200_madgraphMLM-pythia8/GJets-4Jets_dRGJ-0p2_HT-100to200_madgraphMLM-pythia8.json

```diff
@@ -0,0 +1,28 @@
+{
+    "template": "LO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "ickkw": "1",
+        "maxjetflavor": "5",
+        "xqcut" : "10"
+    },
+    "model_params": "massZ.dat",
+    "model_params_user": [
+        "set run_card draj 0.2",
```

Comment:
how did you decide for 0.2? we should make sure we keep track of this as an agreement. what about photon isolation at nlo ?

Reply:
I mean this PR https://github.com/cms-PdmV/GridpackFiles/pull/115#event-10731200208

Reply:
Discussed over skype to keep 0.2

This cut was introduced to regulate the cross section at a reasonable scale so that we can get better affordable statistics. In the past this was done with 0.4 which I thought a bit dangerous for offline analysis so loosened the cut to dR > 0.2. Regarding synchronization between NLO and LO samples, this cut serves slightly different purpose (altho effectively could be similar) so the decision is to do separately.